README.md

```diff
@@ -126,36 +126,31 @@ Usage: kedro viz run [OPTIONS]
 Options:
   --host TEXT               Host that viz will listen to. Defaults to
                             localhost.
-
   --port INTEGER            TCP port that viz will listen to. Defaults to
                             4141.
-
   --browser / --no-browser  Whether to open viz interface in the default
                             browser or not. Browser will only be opened if
                             host is localhost. Defaults to True.
-
-  --load-file FILE          Path to load kedro-viz data from a directory
-  --save-file FILE          Path to save kedro-viz data to a directory 
-  --pipeline TEXT           Name of the registered pipeline to visualise. If not
-                            set, the default pipeline is visualised
-
+  --load-file TEXT          Load Kedro-Viz using JSON files from the specified
+                            directory.
+  --save-file FILE          Save all API responses from the backend as JSON
```

Comment:
since this is more user facing I was thinking maybe we stick to the old 
'Save Kedro-Viz data as JSON files in a specified directory'